database/Images.js

```diff
@@ -2,14 +2,14 @@ const mongoose = require('mongoose');
 const db = require('./index.js');
 
 const imagesSchema = new mongoose.Schema({
-  room_id: Number,
+  roomId: Number,
   title: String,
-  room_photos: Array,
-  host_image: Array,
-  reviewers: Array,
+  roomPhotos: Array,
+  // hostImage: Array,
+  // reviewers: Array,
```

Comment:
out of curiosity: why did you remove these 2?

Reply:
After talking to Amanda, she said that was supose to be part of her second module and she is not using it for her modules